### DISCUSSION TITLE
Everything is hidden?

### DISCUSSION THREAD

**Discussion by noa7777** [keepassxreboot/keepassxc#11187] [Q&A] [answered]
If you enter the content in the input box, the relevant items will still pop up.

**Comment by droidmonkey** [comment]
Excuse me?

**Reply by noa7777** [reply, marked as answer]
![image](https://github.com/user-attachments/assets/26c8d2cc-e7fe-43a0-8b83-de60864ccc01)

**Reply by noa7777** [reply]
![image](https://github.com/user-attachments/assets/ca8ed7b7-e1db-4732-b0fc-a4c6ffe829e3)

**Reply by noa7777** [reply]
But all the entries I saved cannot be edited together and managed intuitively at the same time.

**Reply by droidmonkey** [reply]
https://github.com/keepassxreboot/keepassxc/issues/10927#issuecomment-2180333104

**Reply by noa7777** [reply]
Thank you. I gave it a try and the problem was solved.
